spring-integration-core/src/main/java/org/springframework/integration/graph/Graph.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.graph;
+
+import java.util.Collection;
+
+/**
+ * This object can be exposed, for example, as a JSON object over
+ * HTTP.
+ *
+ * @author Andy Clement
+ * @author Gary Russell
+ * @since 4.3
+ *
+ */
+public class Graph {
+
+	private final Collection<GraphNode> nodes;
+
+	private final Collection<LinkNode> links;
+
+	public Graph(Collection<GraphNode> endpoints, Collection<LinkNode> links) {
```

Comment:
I guess: not `endpoints`.
Or just `nodes`, or `infos` :smile: 

Yes, minor, of course.